micromasters/utils.py

```diff
@@ -19,6 +19,14 @@ def webpack_dev_server_url(request):
     return 'http://{}:{}'.format(webpack_dev_server_host(request), settings.WEBPACK_DEV_SERVER_PORT)
 
 
+def remove_key(dictionary, key):
+    """
+    Helper method to remove a key from a dict and return the dict
+    """
+    del dictionary[key]
+    return dictionary
```

Comment:
this used to be a part of search/api_test.py ([line](https://github.com/mitodl/micromasters/pull/811/files#diff-63f4279d1f66269772e1f02ca030fda9L45)). its being used in a list comprehension, so the function has to return the actual dict after removing the key for it to work correctly. this definitely could use a better name - `remove_key` is (obviously) misleading